app/controllers/appointment_summaries_controller.rb

```diff
@@ -0,0 +1,68 @@
+class AppointmentSummariesController < ApplicationController
+  layout 'guides', only: [:new, :view]
+
+  helper_method :headers
+  helper_method :appointment_type
+
+  def new
+    @appointment_summary = AppointmentSummary.new
+  end
+
+  def view
+    @appointment_summary = AppointmentSummary.new(appointment_summary_params)
+
+    if @appointment_summary.valid?
+      @content = content
+    else
+      render :new
+    end
+  end
+
+  def download
+    @appointment_summary = AppointmentSummary.new(appointment_summary_params)
+    output_document = OutputDocument.new(@appointment_summary, 'generic')
+
+    send_data output_document.pdf,
+              layout: false,
+              filename: 'pension_wise.pdf', type: 'application/pdf'
+  end
+
+  def print
+    @appointment_summary = AppointmentSummary.new(appointment_summary_params)
+    output_document = OutputDocument.new(@appointment_summary, 'generic')
+
+    @content = output_document.html.html_safe
+  end
+
+  private
+
+  def headers(level = 1)
+    content_document.css("h#{level}").each_with_object({}) do |header, headers|
+      headers[header[:id]] = header.text
+    end
+  end
+
+  def content
+    @content ||= Kramdown::Document.new(output_document, input: 'html').to_html.html_safe
+  end
+
+  def output_document
+    @output_document = OutputDocument.new(@appointment_summary).html.html_safe
+  end
+
+  def content_document
+    @content_document ||= Nokogiri::HTML(content)
+  end
+
+  def appointment_summary_params
+    params
+      .require(:appointment_summary)
+      .permit(:supplementary_benefits, :supplementary_debt,
```

Comment:
Could we favour the style where each element, including the opening and closing paren is on a new line? It makes the diff more readable and helps if we add / remove others.